dev/create-release/translate-contributors.py

```diff
@@ -32,14 +32,7 @@
 import sys
 
 from releaseutils import JIRA, JIRAError, get_jira_name, Github, get_github_name, \
-    contributors_file_name, is_valid_author, raw_input, capitalize_author, yesOrNoPrompt
-
-try:
-    import unidecode
-except ImportError:
-    print("This tool requires the unidecode library to decode obscure github usernames")
```

Comment:
Just to clarify, I think now most of release docs, notes, and everywhere else can handle UFT-8 properly, and we don't have to force and mangle other people's name into ASCII. I believe this was initially introduced due to unicode and Python 2's string handling. If there's an issue, we believe we should try to fix it instead of manipulating author's names.